src/applications/hca/validation.js

```diff
@@ -53,3 +53,11 @@ export function validateDependentDate(errors, dependentDate, formData, schema, m
   }
   validateCurrentOrPastDate(errors, dependentDate);
 }
+
+export function validateCurrency(errors, currencyAmount) {
+  // Source: https://stackoverflow.com/a/16242575
+  // HACK: Due to us-forms-system issue 269 (https://github.com/usds/us-forms-system/issues/269)
+  if (!/(?=.*?\d)^\$?(([1-9]\d{0,2}(,\d{3})*)|\d+)?(\.\d{1,2})?$/.test(currencyAmount)) {
```

Comment:
based on my testing this allows spaces before or after the number. Want to make sure we're trimming off whitespace from both sides before this is submitted.

Reply:
Thanks @rianfowler, I was not able to replicate the whitespace issues, but I will include test cases for those.
![image](https://user-images.githubusercontent.com/16051172/45650058-f6dc9980-ba81-11e8-9a6b-6c89eadffa75.png)

Reply:
@cehsu in that second example, is it invalid also because you entered a dollar sign?

Reply:
Dollar signs are allowed per the logic in the CurrecyWidget, but are not stored as formData:
![image](https://user-images.githubusercontent.com/16051172/45715092-353c8c00-bb48-11e8-8db3-2a40235bf50a.png)